docs/scos/dev/glue-api-guides/upcoming-release/managing-carts/guest-carts/managing-guest-carts.md

```diff
@@ -0,0 +1,1239 @@
+<script type="text/javascript" src="https://viewer.diagrams.net/js/viewer-static.min.js"></script>
+
+Below, you can see an exemplary workflow for converting a guest cart into a regular cart:
```

Comment:
Make sense to fix it like in the marketplace doc:
`The following is an exemplary workflow of converting a guest cart into a regular cart:`